hack/functest.sh

```diff
@@ -1,3 +1,8 @@
-#!/bin/bash -e
+#!/usr/bin/env bash
```

Comment:
please keep the previous shebang, this is the one we use all over

plus remove plesase the `set -xe`